RELEASE.md

```diff
@@ -0,0 +1,63 @@
+# Building a new weldr-client release
+
+Make sure you have setup the following in your git environment:
+* user.signingkey
+* user.email
+* user.name
+
+The signingkey is the KEYID of the gpg key you will use to sign the release
+with.  This is used for both the git tag and for the detached signature for the
+source archive file.
+
+Create a branch for the release, this is needed because commits to main are
+restricted to approved pull requests.  eg. `git checkout -b main-release-X.Y`
+
+Run `make release` on this branch.  This will run the tests, bump the version,
+tag the release, and create an archive with a detached gpg signature.  The key
+used to sign the archive will also be output as `gpg-KEYID.key` which will be
+needed for the Fedora release.
+
+Push your branch to GitHub and create a pull request, ask someone for an ack
+if you are not on the list of people who can push to main w/o review.  Also
+wait for the GitHub CI to finish.  Once it has been approved use git on the
+cmdline to do a fast-forward merge into main.  This is so that the signed
+tag matches the commit you made on the release branch.  If GitHub is allowed
```

Comment:
I think the part about fast-forward merging to preserve the commit ID should be emphasised (bold?).  It's a bit important and I can imagine it's easy to accidentally merge through the GitHub UI out of habit.

Reply:
Good idea. Added some bold and broke that out into new paragraphs to make it clearer.